python/batch-predictor/docker/process_conda_env.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+CONDA_ENV_PATH="$1"
+
+echo "Processing conda environment file: ${CONDA_ENV_PATH}"
+echo "Current conda environment file content:"
+cat "${CONDA_ENV_PATH}"
+
+# Remove `merlin-sdk` from conda's pip dependencies
+yq --inplace 'del(.dependencies[].pip[] | select(. == "*merlin-sdk*"))' "${CONDA_ENV_PATH}"
+
+# Add `merlin-batch-predictor` to conda's pip dependencies
```

Comment:
would help to rewrite comment saying if it don't exist merlin-batch-repdictor is pinned to XXX (not immediately obvious this is a conditional replace)